### PR TITLE
Core_Java Assignments-Oops

### PR DESCRIPTION
**Pull request by NehaAug2019**
Part 1 - Complete
part 2 - 7th problem remaining , rest done
Part 3- 1st problem left, rest done

### CODE REVIEW

**Comment by NehaAug2019** [comment]
2 questions left, working on them
Please review rest

File: src/Neha/BankClient.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+c) withdraw()
+d) checkBalance()
+e) displayDetails()
+Create BankClient class to test functionalities of Bank class.
+*/
+package Neha;
+
+public class BankClient {
+	public static void main(String[] args) {
+		Bank b = new Bank();
```
Comment (Harsh9846) [inline src/Neha/BankClient.java]: Hi @NehaAug2019 
here we have to design one system like we provide option like CreditsAmount, DebitAmount
user select any method from switch case or you can give option 
like if select 
1 for --> credits Amount
2 for --> debit Amount etc..

File: src/Neha/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+package Neha;
+
+import java.util.Scanner;
+
+public class Calculator {
+
+	static void cal(int x, int y, String c) {
+		switch (c) {
+		case "+":
```
Comment (Harsh9846) [inline src/Neha/Calculator.java]: always better to write switch case with comment rather than sign.

File: src/Neha/LargestNum.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+//Find the largest of three numbers using Logical And operator.
+package Neha;
+
+import java.util.Scanner;
+
+public class LargestNum {
+	static void largestNum(int x, int y, int z) {
```
Comment (Harsh9846) [inline src/Neha/LargestNum.java]: this simple we can check if already know that z is small than why compare that value again with x and y..
correct that logic.

if( n1>=n2 && n1>=n3)
        println("lf is the largest number."+ n1);
    else if (n2>=n1 && n2>=n3)
        println("lf is the largest number."+ n2);
    else
        println("lf is the largest number."+ n3);

File: src/Neha/Week.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+		default:
+			System.out.println("There are only 7 days in a week");
+
+		}
+	}
+
+	public static void main(String[] args) {
+
+		Scanner sc = new Scanner(System.in);
```
Comment (Harsh9846) [inline src/Neha/Week.java]: what if user wanted know day on 23rd date
user can give any kind input you have handle that number with some logic 

hint: use number 7 and check with user's date
do some operation which gives you some **reminder**

File: src/Neha/BankClient.java (hunk 1 above)
Comment (NehaAug2019) [reply src/Neha/BankClient.java]: @Harsh9846 
This BankClient class is calling Bank class which has all the implementations of switch case

**Pull request merged**